freqtrade/pairlist/PriceFilter.py

```diff
@@ -18,11 +19,17 @@ def __init__(self, exchange, pairlistmanager,
         super().__init__(exchange, pairlistmanager, config, pairlistconfig, pairlist_pos)
 
         self._low_price_ratio = pairlistconfig.get('low_price_ratio', 0)
+        if self._low_price_ratio < 0:
+            raise OperationalException("PriceFilter requires low_price_ratio be >= 0")
         self._min_price = pairlistconfig.get('min_price', 0)
+        if self._min_price < 0:
+            raise OperationalException("PriceFilter requires min_price be >= 0")
         self._max_price = pairlistconfig.get('max_price', 0)
-        self._enabled = ((self._low_price_ratio != 0) or
-                         (self._min_price != 0) or
-                         (self._max_price != 0))
+        if self._max_price < 0:
+            raise OperationalException("PriceFilter requires max_price be >= 0")
+        self._enabled = ((self._low_price_ratio > 0) or
```

Comment:
I think it should also raise OperationalException here (if all settings == 0)

Reply:
i guess so, yeah

Reply:
I guess it depends on how you think people will be using the config. Allowing filters to be disabled means that users can easily leave a filter in their config but run with it disabled, whereas raising an exception will force users to remove the filter from their config.

Reply:
what's the difference to setting all values to 0, vs. putting `//` in front of the filter line ... ?

Reply:
Both the same effect I guess, probably cleaner to enforce the values > 0.